index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+    <main id="main">
+        <h1 id="title">Mahatma Gandhi</h1>
+        <p>Due to Mahatma Gandhi India Came into being</p>
+        <figure id="img-div">
+            <img id="image" src="./images/Mahatma-Gandhi,_studio,_1931.jpg"
+                alt="Mahatma Gandhi Standing" />
+            <figcaption id="img-caption">
+                Mahatma Gandhi Standing
+            </figcaption>
+        </figure>
```

Comment:
I think you should change the **<main>** to **<section>**. and in section you should use **<article>**. In this way will look clean and tidey